web/api/views.py

```
@@ -923,48 +921,45 @@ def post(self, request):
-		if organization_name:
-			organization_obj = None
-			organization_query = Organization.objects.filter(name=organization_name)
-			if organization_query.exists():
-				organization_obj = organization_query[0]
-			else:
-				organization_obj = Organization.objects.create(
-					name=organization_name,
-					project=project,
-					insert_date=timezone.now())
-			organization_obj.domains.add(domain)
+		status = bulk_import_targets(
```

Ruff

`bulk_import_targets` may be undefined, or defined from star imports

(F405)